unison-src/transcripts/ambiguous-metadata.output.md

```diff
@@ -32,7 +32,7 @@ x = 1
   there are multiple matches:
   
     foo.doc
-    boo.doc#tj3gfqdnje
+    doc#tj3gfqdnje
```

Comment:
Not sure why the old code added the extra segment here, or if we want to preserve that behavior... it seems fine?